src/Controls/src/Core/HandlerImpl/SearchBar.Impl.cs

```diff
@@ -0,0 +1,6 @@
+namespace Microsoft.Maui.Controls
+{
+	public partial class SearchBar : ISearchBar
```

Comment:
This and related changes was missed in the original PR and causes a crash. No idea how this passed when I ran it locally.